### PR TITLE
Resolve zustand warnings.

### PR DESCRIPTION
**Pull request by Riyan-Mo**
# Pull Request for Memfree

## Description

Please include a summary of the changes made in this pull request. Explain the purpose of the changes and any relevant context.

## Related Issue

If this pull request addresses an issue, please link to it here. For example:
#87  Fix zustand warning in frontend/lib/store/local-limit.ts.

## Changes Made

- [ ] Added new feature
- [ ] Fixed a bug
- [ ] Updated documentation
- [x] Other (please specify)
Resolved zustand warning

## How to Test

Please describe the steps to test the changes made in this pull request. Include any specific commands or configurations needed.

## Screenshots (if applicable)

If your changes include visual modifications, please add screenshots to help reviewers understand the changes.

## Checklist

- [x] I have read the contributing guidelines
- [x] I have followed the code style guidelines
- [x] My code is tested and works as expected
- [x] I have updated the documentation (if necessary)

## Additional Information

Please provide any additional information that may be helpful for the reviewers.

### CODE REVIEW

File: README.de.md [hunk 1]
```diff
@@ -70,7 +70,11 @@ MemFree ist mit leistungsstarken Funktionen ausgestattet, die verschiedene Such-
 
 -   🌍**Mehrsprachige Unterstützung**: Verfügbar in Englisch, Chinesisch, Deutsch, Französisch, Spanisch, Japanisch und Arabisch.
 
+<<<<<<< HEAD
+-   🔗**Chrome-Lesezeichen-Synchronisierung**: One-click synchronization and indexing.
+=======
 -   🔗**Chrome-Lesezeichen-Synchronisierung**: Synchronisierung und Indizierung mit einem Klick.
```
Comment (ahaapple) [inline README.de.md:76]: There is a conflict

File: README.zh-CN.md [hunk 2]
```diff
@@ -8,7 +8,11 @@
 
 <div align="center">
 
-**英语**\|[中文](./README.zh-CN.md)\|[德语](./README.de.md)\|[法语](./README.fr.md) \| [西班牙语](README.es.md)\|[日本人](./README.ja.md)
+<<<<<<< HEAD
+**英语**\|[中文](./README.zh-CN.md)\|[德语](./README.de.md)\|[法语](./README.fr.md)\|[西班牙语](README.es.md)\|[日本人](./README.ja.md)
```
Comment (ahaapple) [inline README.zh-CN.md:12]: There is a conflict

**Comment by Riyan-Mo** [comment]
I have resolved the conflict but it is asking me to create a new pull request from the same branch.

**Comment by ahaapple** [comment]
@Riyan-Mo you only need to push the new commit to zustand-warnings branch, don't need to create a new PR.

**Pull request closed without merging**